packages/dds/tree/src/index.ts

```diff
@@ -315,6 +312,12 @@ export {
 	NodeBuilderDataUnsafe,
 	NodeFromSchemaUnsafe,
 
+	// experimental @internal APIs:
+	adaptEnum,
```

Comment:
Not for this PR, but I'm curious: when do you think the enum-related APIs here will be ready for beta testing?

Reply:
I was gonna do it just after this one: I'd forgotten about that until I made this change. I'd do it now, but that would cause conflicts.